### DISCUSSION TITLE
Date format in exported highlights

### DISCUSSION THREAD

**Discussion by ichnilatis-gr** [koreader/koreader#8327] [Q&A]
Is there any way to change the format of the dates in the html file of the extracted highlights? I.e. from MM/DD/YYYY to DD/MM/YYYY.

Thanks!

**Comment by Frenzie** [comment]
It seems to be hard-coded to the locale, which might not be of much use on most embedded devices.
https://github.com/koreader/koreader/blob/a1e0d87c62ddad84bb6a422cd3714aa02d6833ac/plugins/exporter.koplugin/note.tpl#L60

Don't we have some kind of date display setting elsewhere these days? It can probably be reused here if so.
